<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-LOSCQCA" failures="1" tests="3" name="Test" time="1.157" errors="0" timestamp="2023-10-15T12:31:33 IST">
  <testcase classname="extentReportBySarthak.AddToCartTest" name="addToCartTest" time="0.014"/>
  <testcase classname="extentReportBySarthak.LoginTest" name="loginTest" time="0.002"/>
  <testcase classname="extentReportBySarthak.PaymentTest" name="loginTest" time="0.019">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at extentReportBySarthak.PaymentTest.loginTest(PaymentTest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Test -->
